kousa/priv/repo/migrations/20210305231415_notifications_table.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Beef.Repo.Migrations.NotificationsTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:notifications, primary_key: false) do
+      add :id, :uuid, primary_key: true, default: fragment("uuid_generate_v4()")
+      add :type, :string, null: false
+      add :is_read, :boolean, default: false
+      add :user_id, references(:users, on_delete: :delete_all, type: :uuid), null: false, primary_key: true
+      add :notifier_id, references(:users, on_delete: :delete_all, type: :uuid), null: true, primary_key: true
+      timestamps()
```

Comment:
we switched to `:utc_datetime_usec` do:

```
      add :inserted_at, :utc_datetime_usec, null: false, default: fragment("now()")
      add :updated_at, :utc_datetime_usec, null: false, default: fragment("now()")
```
instead of `timestamps()`